.gitignore

```diff
@@ -12,3 +12,4 @@ log-camel-lsp.out
 uitest-resources
 test Fixture with speci@l chars/.vscode/**
 test-extensions
+coverage/
```

Comment:
better if extra end of line is added at the end of files. it allows better diff on next modifications of the files